### PR TITLE
Register bimarshaw.is-a.dev

### PR DESCRIPTION
**Pull request by Bimarshaaa**
Added `bimarshaw.is-a.dev` using the [CLI](https://www.npmjs.com/package/@is-a-dev/cli).

### CODE REVIEW

File: domains/bimarsha.is-a.dev.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "owner": {
+        "username": "Bimarsha69",
+        "email": "bimarshathapa2@gmail.com"
+    },
+
+    "record": {
+        "A": ["bimarsha.live"]
```
Comment (DEV-DIBSTER) [inline domains/bimarsha.is-a.dev.json:8]: Invalid A record. A records must be IPv4.

File: domains/bimarsha.is-a.dev.json
Comment (DEV-DIBSTER) [inline domains/bimarsha.is-a.dev.json:1]: Domain name format is wrong.

File: domains/bimarsha.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+    "description": "Welcome To Bimarshaˈs World",
+    "repo": "https://github.com/Bimarsha69/bimarsha.github.io",
+    "owner": {
+        "username": "Bimarsha69",
+        "email": "bimarsha@bimarshathapa.com.np"
+    },
+    "record": {
+        "CNAME": "bimarsha.github.io"
```
Comment (DEV-DIBSTER) [inline domains/bimarsha.json:9]: 404 CNAME record. Please provide a valid CNAME record.

File: domains/bimarshaa.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{
+    "owner": {
+        "username": "Bimarsha69",
+        "email": "bimarshathapa2@gmail.com"
+    },
+
+    "record": {
+        "A": ["bimarsha.me"]
```
Comment (DEV-DIBSTER) [inline domains/bimarshaa.json:8]: Invalid A record. A records must be IPv4.

File: domains/bimarshat.json [hunk 4]
```diff
@@ -0,0 +1,10 @@
+{
+    "owner": {
+        "username": "Bimarsha69",
+        "email": "bimarshathapa2@gmail.com"
+    },
+
+    "record": {
+        "A": ["bimarsha.me"]
```
Comment (DEV-DIBSTER) [inline domains/bimarshat.json:8]: Invalid A record. A records must be IPv4.

File: domains/bimarshaw.json [hunk 5]
```diff
@@ -0,0 +1,10 @@
+{
+    "owner": {
+        "username": "Bimarsha69",
+        "email": "bimarshathapa2@gmail.com"
+    },
+
+    "record": {
+        "A": ["info.bimarshathapa.com.np"]
```
Comment (DEV-DIBSTER) [inline domains/bimarshaw.json:8]: Invalid A record. A records must be IPv4.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

File: domains/bimarsha.is-a.dev.json
Comment (WilliamDavidHarrison) [reply domains/bimarsha.is-a.dev.json:1]: To add to DIBSTER's review, remove the `.is-a.dev` part.

File: domains/bimarshaw.json (hunk 5 above)
Comment (WilliamDavidHarrison) [reply domains/bimarshaw.json:8]: Please use a CNAME instead.

File: domains/bimarshat.json (hunk 4 above)
Comment (WilliamDavidHarrison) [reply domains/bimarshat.json:8]: Please use a CNAME instead.

File: domains/bimarsha.is-a.dev.json (hunk 1 above)
Comment (WilliamDavidHarrison) [reply domains/bimarsha.is-a.dev.json:8]: Please use a CNAME instead.

**Pull request closed without merging**